lib/testing/src/lib/core/pages/login.page.ts

```diff
@@ -183,7 +183,7 @@ export class LoginPage {
 
     goToLoginPage() {
```

Comment:
enableFooter, disableFooter and others are methods specific to demo-shell, not needed in the generic page object